### PR TITLE
Fix emtest

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1-added functionality as script able to calculate system frequency threshold
so automated run  works well
2-change test cancel to fail as if frequency scaling if not fit
in the threshold, test should fail
3- fix pep8 issue in em_quad_level_frequency_transitions 

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
Job log:
```
[root@system cpu]# avocado run em_quad_level_frequency_transitions.py
JOB ID     : 94fb5e3c5f4f888c2bae96c6709158ac12253e00
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2019-10-23T04.08-94fb5e3/job.log
 (1/1) em_quad_level_frequency_transitions.py:freq_transitions.test: PASS (412.05 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 412.73 s
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2019-10-23T04.08-94fb5e3/results.html
[root@system cpu]# 
```

File: cpu/em_quad_level_frequency_transitions.py [hunk 1]
```diff
@@ -61,6 +62,19 @@ def setUp(self):
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
 
+        fre_min = 0
+        fre_max = 0
+        freq_info = process.system_output(
+            "cpupower frequency-info", shell=True).decode("utf-8")
+        for line in str(freq_info).splitlines():
+            if re.search('hardware limits:', line, re.IGNORECASE):
```
Comment (narasimhan-v) [inline cpu/em_quad_level_frequency_transitions.py]: are we looking at one of multiple such matching lines ?

Comment (narasimhan-v) [reply cpu/em_quad_level_frequency_transitions.py]: Also, do we really need `re` here ? `if 'hardware limits:' in line:` won't work ?

**Comment by PraveenPenguin** [comment]
@narasimhan-v please have a look on this PR

**Comment by PraveenPenguin** [comment]
merging @narasimhan-v thanks for ACK

**Pull request merged**